runtime/lua/vim/lsp/util.lua

```diff
@@ -577,28 +577,28 @@ end
 
 --- Parses snippets in a completion entry.
 ---
---@param input (string) unparsed snippet
---@returns (string) parsed snippet
+---@param input (string) unparsed snippet
+---@returns (string) parsed snippet
 function M.parse_snippet(input)
   local res, _ = parse_snippet_rec(input, false)
 
   return res
 end
 
---@private
+---@private
 --- Sorts by CompletionItem.sortText.
 ---
---@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_completion
+--see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_completion
```

Comment:
Shouldn't this be 
```suggestion
---@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_completion
```

Reply:
I can update these, but since they weren't already included in the docs I chose to err toward leaving it how it is.